.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Feature request
+about: Thanks for taking the time to fill out this feature request! Please complete
+  the following form to help us assist you.
+title: "[FEAT] "
+labels: enhancement
+assignees: yuvneshtruera
+
+---
+
+**Feature Description**
+Describe the feature you are requesting. Try to reference existing implementations/papers/examples when possible.
+
+**Reason**
+What is stopping TruLens from supporting this feature today? What existing approaches have not worked for you?
```

Comment:
I don't think we should write it like this, maybe try this "Why is this needed? Is there a workaround or another product you are using today to fulfill this gap?"